portfolio/src/main/webapp/createpuzzle.html

```diff
@@ -27,8 +27,12 @@
         <div class="container w-100 py-4">
             <div class="container w-25 py-4">
                 <div id="upload-message" class="pb-4 d-none"></div>
-                <!-- Form for users to send their image and select the difficulty of the puzzle-->
+                <!-- Form for users to send their image and select the difficulty of the puzzle-->                
                 <form id="create-puzzle-form" action="/upload" method="POST">
+                    <div class="mb-3">
+                        <label for="name">Enter Your Name</label>
```

Comment:
I was thinking more along the lines of a name to describe the image being uploaded